nmpolicy/internal/resolver/resolver_test.go

```diff
@@ -51,6 +51,7 @@ routes:
     table-id: 254
 interfaces:
   - name: eth1
+    description: "This is eth1"
```

Comment:
You use it several times in the tests, maybe move it to a const as well.
At the same time, maybe make it nicer, like: "1st ethernet interface" or something like that.